src/client/pages/Checkout/CheckoutDetails/components/YourPlan/YourPlan.tsx

```diff
@@ -65,13 +58,13 @@ export const YourPlan = () => {
   return (
     <SubSection headlineText={textKeys.CHECKOUT_INSURANCE_TITLE()}>
       <>
-        {quotes.map(({ displayName, price }) => (
-          <Row key={displayName}>
-            <Label>{displayName}</Label>
-            <Value>{formattedPrice(price.amount)}</Value>
+        {prices.map((item: Price) => (
+          <Row key={item.displayName}>
+            <Label>{item.displayName}</Label>
+            <Value>{formattedPrice(item.price)}</Value>
           </Row>
         ))}
-        {Number(discount) > 0 && (
+        {Number(discount) > 0 && discount && (
```

Comment:
Shouldn't this be:

```suggestion
        {discount && Number(discount) > 0 && (
```

Why isn't it being passed as a bumber to the component?